cmd/fyne/vendor.go

```diff
@@ -22,14 +22,17 @@ const (
 	goModFile = "go.mod"
 	// goModVendorFile is the vendor module file
 	goModVendorFile = "vendor/modules.txt"
-	// glfwModNew is the glfw module regexp for 3.3+
-	glfwModNew = `github\.com/go-gl/glfw/(v(\d+\.)?(\*|\d)?(\*|\d+))/glfw`
-	// glfwModOld is the glfw module regexp for 3.2
-	glfwModOld = "github.com/go-gl/glfw"
 	// glfwModSrcDir is the glfw dir containing the c source code to copy
 	glfwModSrcDir = "glfw"
 )
 
+var (
+	// glfwModNew is the glfw module regexp for 3.3+
+	glfwModNew = regexp.MustCompile(`github\.com/go-gl/glfw/(v(\d+\.)?(\*|\d)?(\*|\d+))/glfw`)
+	// glfwModOld is the glfw module regexp for 3.2
+	glfwModOld = regexp.MustCompile(`github\.com/go-gl/glfw`)
```

Comment:
Does this need to be a regexp? Seems like a simple string comparison would work as there is no number matching

Reply:
It isn't a simple string comparison. It is checking the anywhere in the input because of not having start and end characters. I am just fixing up the obvious things here 🙂

Reply:
Ah okay, well `strings.Contains` may still be more efficient, tho I appreciate that you're trying to minimize the size of this code change